lib/qesapdeployment.pm

```diff
@@ -74,6 +77,7 @@ sub qesap_get_file_paths {
     $paths{deployment_dir} = get_var('QESAP_DEPLOYMENT_DIR', get_var('DEPLOYMENT_DIR', '/root/qe-sap-deployment'));
     $paths{terraform_dir} = get_var('PUBLIC_CLOUD_TERRAFORM_DIR', $paths{deployment_dir} . '/terraform');
     $paths{qesap_conf_trgt} = $paths{deployment_dir} . "/scripts/qesap/" . $paths{qesap_conf_filename};
+    $paths{qesap_conf_src} = data_url("sles4sap/qe_sap_deployment/" . $paths{qesap_conf_filename});
```

Comment:
Coding style consistent: using ' instead of "

Reply:
Thanks. Will be fixed in next push